recipes/nglview/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: nglview
+  version: "0.5.dev"
+
+source:
+  git_url: https://github.com/arose/nglview
```

Comment:
You can also pin a version specific to a commit, but I'm sure Alex will release a version soon :)
ping @arose